EasyPost/Models/API/Message.cs

```diff
@@ -8,13 +8,13 @@ public class Message : EasyPostObject
         #region JSON Properties
 
         [JsonProperty("carrier")]
-        public string carrier { get; set; }
+        public string? Carrier { get; set; }
         [JsonProperty("carrier_account_id")]
-        public string carrier_account_id { get; set; }
+        public string? CarrierAccountId { get; set; }
         [JsonProperty("message")]
-        public string message { get; set; }
+        public string? Text { get; set; } // "Message" is the enclosing class name
```

Comment:
Please call this and any other complete name changes out in the CHANGELOG

Reply:
Done